src/pages/panel/views/simple.js

```diff
@@ -20,37 +20,38 @@ import {
 import sites from '/rule_resources/sites.json';
 
 import Stats from '/store/stats.js';
-import Rulesets, { rulesetIds } from '/store/rulesets.js';
+import Options from '/store/options.js';
 
 import Detailed from './detailed.js';
 
 function toggleRuleset(ruleset) {
   return (host) => {
-    store.set(host.rulesets, {
-      [ruleset]: !host.rulesets[ruleset],
+    store.set(host.options, {
+      dnr: {
+        [ruleset]: !host.options.dnr[ruleset],
+      },
     });
   };
 }
 
 export default define({
   [router.connect]: { stack: [Detailed] },
   tag: 'gh-panel-simple-view',
-  rulesets: store(Rulesets),
+  options: store(Options),
   stats: store(Stats),
-  render: ({ rulesets, stats }) => html`
+  render: ({ options, stats }) => html`
     <h1>${t('privacy_protection')}</h1>
 
     <section class="toggles">
-      ${store.ready(rulesets) &&
-      rulesetIds.map(
+      ${store.ready(options) &&
+      Object.keys(options.dnr).map(
```

Comment:
is the order of keys always fixed?

Reply:
Yes, it is written in the ES specification. It always follows the order of how the keys are defined.